### PR TITLE
Update Install.md file

### PR DESCRIPTION
**Pull request by SHR1SHAK**

### CODE REVIEW

File: static/images/Install/Installation-completion.png
Comment (CodingWonders) [inline static/images/Install/Installation-completion.png:1]: Can you enlarge it?

File: static/images/Install/Installation.png
Comment (CodingWonders) [inline static/images/Install/Installation.png:1]: Can you enlarge it?

File: static/images/Install/Select-Apps-to-install-upgrade.png
Comment (CodingWonders) [inline static/images/Install/Select-Apps-to-install-upgrade.png:1]: Can you enlarge it?

File: static/images/Install/track-progress.png
Comment (CodingWonders) [inline static/images/Install/track-progress.png:1]: Can you enlarge it?

**Review by CodingWonders — CHANGES_REQUESTED** [review]

File: static/images/Install/Installation-completion.png
Comment (SHR1SHAK) [reply static/images/Install/Installation-completion.png:1]: sure

**Comment by SHR1SHAK** [comment]
Fixed the issue. Please check

**Review by CodingWonders — APPROVED** [review]

**Comment by SHR1SHAK** [comment]
Fixed the uncropped image

**Pull request merged**